### ISSUE TITLE
British technical grades

### ISSUE THREAD

**Issue by musoke** [OpenBeta/sandbag#170]
Add [British technical grades](https://en.wikipedia.org/wiki/Grade_(climbing)#British_E-grade)

These are the ones that look like French sport grades, not E-grades.

**Comment by Agilulfo** [comment]
I think this is a quite complex topic, since this trad scale takes into account many aspects beside the difficulty. 
For instance it takes into account the quality of protection and other factors.
All the other sport scales I know are focused only on the difficulty.
I suspect you cannot implement such type of conversion.

**Comment by musoke** [comment]
Yes, I don't anticipate being able to convert British trad grades usefully.  At best we would be converting to quite wide grade ranges.

More important is being able to parse the grades, so that British trads can be listed on openbeta.io.
